Multisig/UI/Assets/Collectibles/CollectiblesViewModel.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  CollectiblesViewModel.swift
+//  Multisig
+//
+//  Created by Moaaz on 7/8/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+class CollectiblesViewModel: BasicLoadableViewModel {
```

Comment:
CollectibleListViewModel?
It is easy to confuse CollectibleViewModel and CollectiblesViewModel